tools/git-hooks/pre-commit

```diff
@@ -4,3 +4,4 @@
 set -e
 
 ./tools/git-hooks/lint.sh
+./tools/git-hooks/build.sh
```

Comment:
Since I'm gonna publish this at one point I don't want to commit the build artifacts (they would only be useful for people installing from `git://` addresses).